lib/TelegramHandlers/TgPhotoHandler.js

```diff
@@ -0,0 +1,113 @@
+const TgHelpers = require('./TgHelpers.js');
+const Helpers = require('../Helpers.js');
+
+/**
+ * Handles the event when a user sends a photo to the Telegram Group.
+ */
+class TgPhotoHandler {
+
+    // ---------------- Constructor ----------------
+
+    /**
+     * 
+     * @param {boolean} enabled - Is this handler enabled?
+     * @param {*} tgbot - Reference to the Telegram Bot API.
+     * @param {*} action - The action to take when this handler is fired.
+     *                     Only parameter is a string, which is the message
+     *                     to send out.
+     */
+    constructor(enabled, tgbot, action) {
+        this.Enabled = enabled;
+        this._tgbot = tgbot;
+        this._action = action;
+    }
+
+    // ---------------- Functions ----------------
+
+    /**
+     * 
+     * @param {*} from - Object that contains the information about the user name.
+     * @param {*} photoList - The information about the photos we can get from Telegram.
+     * @param {string} - The caption of the photo, if any.
+     */
+    async ReportPhoto(from, photoList, caption) {
```

Comment:
How about `RelayPhoto` or `SendPhoto`? `ReportPhoto` makes me think of a moderation-type function like `Report Abuse`.

Reply:
All better.